### PR TITLE
Work section

### PR DESCRIPTION
**Pull request by Ngufor-emmanuella**
In this project I created a portfolio showing my skillls in
-HTML as the language
-used CSS to design my webpage
-used css grid and flex box to align my elements 
-and linters to check my codes

### CODE REVIEW

**Review by danielufeli — CHANGES_REQUESTED** [review]
Hi @Ngufor-emmanuella , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has an unprofessional readme, Kindly update your readme to be more professional using the [Readme Template](https://github.com/microverseinc/readme-template)

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: style.css [hunk 1]
```diff
@@ -61,22 +66,249 @@ p {
...
+  row-gap: 30px;
+  background: rgb(245, 245, 245);
+}
+
+.work-container .second {
+  background: white;
+  margin: 15px;
+  border-radius: 15px;
+  border: 1px #dfe1e6 solid;
+}
```
Comment (Kakalanp) [inline style.css:305]: Try aligning the items inside the _second_ class with flexbox :nerd_face:

File: index.html [hunk 2]
```diff
@@ -32,20 +33,212 @@ <h5>LET'S CONNECT</h5>
...
+                        <ul class="pro">
+                            <li class="js"><img src="images/Ellipsejs.png" alt="icons"> javascript</li>
+                            <li class="js"><img src="images/Ellipsehtml.png" alt="icons"> HTML</li>
+                            <li class="js"><img src="images/Ellipsecss.png" alt="icons">CSS</li>
+                    </ul>
+                    <ul class="fra">
+                        <li class="fro"><h2>Frameworks </h2><i class="fas fa-angle-right"></i></li>
+                        <li class="fro"><h2>Skills </h2><i class="fas fa-angle-right"></i></li>
+                    </ul>
+                </div>
```
Comment (Kakalanp) [inline index.html:211]: Great work of `<ul>` and `<li>` on frameworks and skill, but remember languages is also part of that list (a nested list), so kindly add it to the `<ul>` as a `<li>` too :fire:

File: style.css (hunk 1 above)
Comment (Kakalanp) [reply style.css:305]: Also, a recommendation to keep your design closer to the Figma version could be:
```
margin: 15px auto;
max-width: 90%;
```

File: style.css [hunk 3]
```diff
@@ -1,26 +1,27 @@
...
-    box-sizing: border-box;
+  background-image: url("images/headershapmobil@2x.svg");
+  background-repeat: no-repeat;
+  background-size: cover;
+  width: auto;
+  height: 100%;
+  font-family: 'Poppins', sans-serif;
+  box-sizing: border-box;
+  overflow: scroll;
 }
```
Comment (Kakalanp) [inline style.css:15]: [OPTIONAL]

Your background image seems to make this paragraph difficult to read 
image

I'd advice you to add the background image to the container of your presentation to fix it :wink:

**Review by Kakalanp — CHANGES_REQUESTED** [review]
# Hello there @Ngufor-emmanuella! hi

### Good job so far! :tada: 

You still need to work on some issues to go to the next project, but you are almost there! :pinching_hand: 

so close

## Highlights :+1: 
- No linter errors :heavy_check_mark:
- Repo looks professional :briefcase:
- Solid pull request :scroll:
- Mostly followed best practices :monocle_face: 

## Required Changes ♻️
***Check the comments under the review.***

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 4]
```diff
@@ -1,7 +1,6 @@
 # Project Name 
 First-micro-project
```
[the remark this answers is not in the document]
Comment (Kakalanp) [reply README.md]: Great work fixing this one while I finished your review! :smiley:

**Comment by Ngufor-emmanuella** [comment]
thank you

**Pull request merged**